src/server/json_family.cc

```diff
@@ -490,8 +491,66 @@ OpResult<long> OpDel(const OpArgs& op_args, string_view key, string_view path) {
   return total_deletions;
 }
 
+OpResult<vector<StringVector>> OpObjKeys(const OpArgs& op_args, string_view key,
```

Comment:
maybe worth adding a comment explaining what `vector<StringVector>` corresponds to.